assets/script.js

```diff
@@ -0,0 +1,46 @@
+const adviceAPIUrl = 'https://api.adviceslip.com/advice';
+const adviceId = document.querySelector('#adivceId');
```

Comment:
```suggestion
const adviceIdElement = document.querySelector('#adivceId');
```
Optional, dar recomandat - sa denumim variabilele cat mai sugestive - mai ales daca sunt variabile globale (nu doar locale) ca sa stim ce sunt ele cand le folosim in alte parti, inclusiv functii